unsplash.gemspec

```diff
@@ -18,10 +18,11 @@ Gem::Specification.new do |spec|
   spec.executables   = spec.files.grep(%r{^exe/}) { |f| File.basename(f) }
   spec.require_paths = ["lib"]
 
-  spec.required_ruby_version = ">= 2.3.0"
+  spec.required_ruby_version = ">= 2.7.0"
 
-  spec.add_dependency "httparty", "~> 0.16"
-  spec.add_dependency "oauth2",   "~> 1"
+  spec.add_dependency "httparty", "~> 0.20"
+  spec.add_dependency "oauth2",   ">= 2.0.8"
+  spec.add_dependency "faraday-multipart", "~> 1.0.4"
```

Comment:
Faraday's multipart middleware was extracted to a separate gem: https://github.com/lostisland/faraday/pull/1357